brainstorm/src/components/items.tsx

```diff
@@ -78,15 +98,14 @@ export function ItemsView(props: {
 	}
 }
 
-/**
- * Removes a node from its parent {@link Items}.
- * If the note is not in an {@link Items}, it is left unchanged.
- */
-export function deleteItem(item: Item): void {
+export interface ItemParent {
+	deleteItem(item: Item): void;
+}
+
+export function deleteItemFromParent(item: Item & TreeNode): void {
 	const parent = Tree.parent(item);
-	// Use type narrowing to ensure that parent is Items as expected for an Item.
 	if (Tree.is(parent, Items)) {
```

Comment:
the main reason we are going this change is to remove this reference to Items. You need to actually remove it. Instead try and read the symbol off the parent